src/services/register.location.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Resource } from "@companieshouse/api-sdk-node";
+import { ApiErrorResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
+import { Session } from "@companieshouse/node-session-handler";
+import { RegisterLocation, ConfirmationStatementService } from "private-api-sdk-node/dist/services/confirmation-statement";
+import { createPrivateOAuthApiClient } from "./api.service";
+
+export const getRegisterLocationData = async (session: Session, companyNumber: string): Promise<RegisterLocation[]> => {
+  const client = createPrivateOAuthApiClient(session);
+  const csService: ConfirmationStatementService = client.confirmationStatementService;
+  const response: Resource<RegisterLocation[]> | ApiErrorResponse = await csService.getRegisterLocations(companyNumber);
+  const status = response.httpStatusCode as number;
+  if (status >= 400) {
+    const errorResponse = response as ApiErrorResponse;
+    throw new Error(`Error retrieving register location data from confirmation-statment api: ${JSON.stringify(errorResponse)}`);
```

Comment:
use 
throw createAndLogError(`Error retrieving register location data from confirmation-statment api: ${JSON.stringify(errorResponse)}`);
instead, it will log the error as well.
from "../../utils/logger"